### PR TITLE
Add Amazon SES provider docs

### PR DESCRIPTION
**Pull request by deandre**
## Change description

> Add Amazon SES provider docs

## Type of change

- [ ] Bug (fixes an issue)
- [x] Enhancement (adds functionality)

## Related issues

> Fix [MAG-572](https://linear.app/magicbell/issue/MAG-572/new-amazon-ses)

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: docs/email/amazon-ses.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        }
+      }
+    }
+  }
+}
+```
+
+## Using Amazon SES email templates
+
+MagicBell can deliver email notifications using Amazon SES email templates.
```
Comment (unamashana) [inline docs/email/amazon-ses.mdx:57]: Good to mention, and link to 'You can use our templates, or use Amazon's',

File: docs/email/amazon-ses.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+<img
+  className="full-mobile"
+  src="/docs/uploads/email_amazon-ses.gif"
+  style={{ width: '100%', height: 'intrinsic' }}
+/>
+
+<Note>You can optionally configure a custom from email address.</Note>
+
+Once your account is set up, MagicBell will automatically deliver email notifications through your Amazon SES account, **no additional work is needed**.
```
Comment (unamashana) [inline docs/email/amazon-ses.mdx:22]: Should we mention that you should verify this email address in AWS before setting it in MB?

**Review by unamashana — APPROVED** [review]

File: docs/email/amazon-ses.mdx (hunk 1 above)
Comment (deandre) [reply docs/email/amazon-ses.mdx:57]: @unamashana I'll create a new issue/PR for this as it should be updated for all providers.

Comment (deandre) [reply docs/email/amazon-ses.mdx:63]: Addressed this in https://github.com/magicbell-io/docs/pull/147!

**Pull request merged**